src/Controllers/GagePageController.ts

```diff
@@ -966,144 +1132,291 @@ module StreamStats.Controllers {
             });
         }
 
-        public createDailyRasterPlot(): void {
-            if (this.dailyValuesOnly.length > 0) {
-                // sort array ascending
-                const asc = this.dailyValuesOnly.sort((a, b) => a - b);
-                //caluculate percentile values
-                var fifthPercentile = asc[Math.floor(asc.length * 0.05)];
-                var ninetyfifthPercentile = asc[Math.floor(asc.length * 0.95)];
-            };
-            function logOrLinear(dailyValuesOnly) {
-                if (dailyValuesOnly.some(v => v <= 0)) {
-                    return {
-                        type: 'linear',
-                        min: fifthPercentile, 
-                        max: ninetyfifthPercentile,
-                        stops: [                     
-                        [0 ,   '#FF0000'],
-                        [0.3, '#FFCC33'],
-                        [0.8, '#66CCFF'],
-                        [1 ,   '#3300CC']
-                        ],
-                        startOnTick: false,
-                        endOnTick: false,
-                        labels: {
-                            format: '{value} ft³/s'
-                        },
-                        allowNegativeLog: true
+public getCorrectColor(date): string {
+    let days = (new Date().getTime() - new Date(date).getTime()) / (1000 * 60 * 60 * 24)
+        if (days <= 31) {
+            console.log("first month", days)
```

Comment:
We try to clean up console.logs before we merge. This just cleans up the console for the next person working.